circle.yml

```diff
@@ -14,6 +14,13 @@ dependencies:
   cache_directories:
     - "~/.cabal"
 
+  override:
+    - mkdir -p /home/ubuntu/.go_workspace/src/github.com/Originate
+    - cp -r /home/ubuntu/git-town /home/ubuntu/.go_workspace/src/github.com/Originate/gt
+    - cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt && go install
+    - rm -rf /home/ubuntu/.go_workspace/src/github.com/Originate/gt
```

Comment:
Circle caches the Go workspace. No point in caching this directory here, since it is getting copied over at the next test run. Removing it also prevents a number of issues with the `cp` command above, like properly removing files here when they get deleted or renamed in the source code.